meshtastic/util.py

```diff
@@ -25,14 +25,16 @@
 from meshtastic.version import get_active_version
 
 """Some devices such as a seger jlink or st-link we never want to accidentally open
-0x1915 NordicSemi (PPK2)
+     1915 NordicSemi (PPK2)
+     0925 Lakeview Research Saleae Logic (logic analyzer)
+04b4:602a Cypress Semiconductor Corp. Hantek DSO-6022BL (oscilloscope)
 """
-blacklistVids = dict.fromkeys([0x1366, 0x0483, 0x1915])
+blacklistVids = dict.fromkeys([0x1366, 0x0483, 0x1915, 0x0925])
 
 """Some devices are highly likely to be meshtastic.
 0x239a RAK4631
 0x303a Heltec tracker"""
-whitelistVids = dict.fromkeys([0x239a, 0x303a])
+whitelistVids = dict.fromkeys([0x239a, 0x303a, 0x04b4])
```

Comment:
What device is `0x04b4`? Probably worth documenting so we don't lose what these are, I think

Reply:
heh.  I don't know I'll look around.  I had the same thought when I added my second vid ;-).  I just tried plugging in a few different boards and didn't see it I'll check the commit history and hopefully I had a comment. ;-)